src/view/seed.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { View, StyleSheet } from 'react-native';
+import { observer } from 'mobx-react';
+import PropTypes from 'prop-types';
+import MainContent from '../component/main-content';
+import { CopyText, H1Text, Text } from '../component/text';
+import { SplitBackground } from '../component/background';
+import { GlasButton } from '../component/button';
+import { List } from '../component/list';
+import { color, font } from '../component/style';
+
+const styles = StyleSheet.create({
+  splitTop: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+  title: {
+    padding: 20,
+    fontSize: font.sizeXXL,
+    textAlign: 'center',
+  },
+  copyTxt: {
+    alignSelf: 'center',
+    textAlign: 'center',
+    fontSize: font.sizeL,
+    lineHeight: font.lineHeightL,
+    paddingLeft: 20,
+    paddingRight: 20,
+    paddingBottom: 20,
+    maxWidth: 600,
+  },
+  wordList: {
+    flexDirection: 'row',
+    flexWrap: 'wrap',
+    justifyContent: 'center',
+    alignItems: 'center',
+    paddingTop: 25,
+    paddingBottom: 25,
```

Comment:
I'm wondering if it's possible to have the List float in the center of the MainView instead of adjusting the position via paddingTop?